### PR TITLE
UnionPay QuickPass Logo

### PR DESCRIPTION
**Pull request by LaNeraNura**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @LaNeraNura ,

There seems to be SVG markup error. I can't view the file.

Can you please fix the errors?

**Comment by LaNeraNura** [comment]
Hi Thomas, 
I have recreated the svg file. Kindly take a look if this format can be viewed now.
Thanks!

File: app/assets/images/payment_icons/unionpay_quickpass.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2OCup6MmILtepEpuhC06LLtgNqoCkIchCkE4/ XKIABgcpz2ZFAZzerVUOWCXbsiO2fz4IjA1a1QHWCLCw0mdXMZSsxlPC6to/0tBpDevoTiH/bpz7 zQeqskXFBysutcRe4ss2yTXJt8CYK7JeF2LXYGrwoOmKh7JAT06Sevf09rangH0ZsM4+FAb7gMwg kDtWwcpViPSEVY5ubx/t1o1zPzpVzKMlYuiZMFw7FJKccFgQGqIimx3T1JaYKiTiqpqIq3rUMBGT GYsKEEIinKDgsKBAoUDF0ynfbYIMkd74k57twjMGi6N4xUnLKk+MMLmGVAJ4/vvwZHtembz+xJ06 WsOqfACmAas2ZaJWwD5ixIqaPDdVuIwvtvxUXCB3HhcOI84qxjUrR2CuKwgZSxQytiym8zIGbEVM 9eWGBvqz6b4hx+07NJA7aEPeV4DbBehDNuS0BWfIwtDgyXDqV6nEviMBq/QlyXNeiUYjqiBE5Ww2 EjBNIylDDzMYQUVqbE3WNUvMblQEIS4zMawJYlQDDEUQVUGgMxNBFgRXIX5BURQlQjJBcMmrarhv ykFqlBJ2hFvCRNuyI7+rdUtUK+CMxPBXKjre9WKNcVHv9eeP7vUSzZuWSodSzBIpHHMs0XVcJlgm E+2C4DoWs00bQtZhGDBFNlQQ3P68iM49HYcPDznoTAPZApBPqWqWicagqLEBa2Bg8Hh7plcnVePX escDVvmr0dbxrEgkIplmVHXdsCWZYYFBFxmnBRRlTiIoBTRRDEkOjLp43JBcUPqegCAwRRKIEZBA bGQmUnaW0lIKO+Pv6FxA9OPjxx+AGneEqHVLWbO1olYJqk3BqT2O9EQ//wjREDmXPifnFZjNmGg5 YJYtCjlXZJmu/v6sDSmbF5y0LSBH53887YgAy7SRZyKGFBFDTh4DrwOD75St3lSBrVZxm+pzjvd1 /nvx3ysBOQgYRgBh0UFQZQhIDIYGOSDbtqpCUgVAESRJkuCIgkjeeV7xDPIjpVxHKidGG3315IBV a8dQopzp8i7sAcqK7cBxJO426uQBy5FhMgGmIyCXF5BlEnKujUxWw1AmhexDPqX9iNG8Vjg+6Tr2 /wMy/Z8+kHigBAAAAABJRU5ErkJggg==" id="image902" opacity="1" stop-opacity="1"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/unionpay_quickpass.svg:1]: Can we remove unused `id` attributes if it's not used? I see some instances:

- `path834`
- `path836`

Also is the `<defs></defs>` needed?

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by LaNeraNura** [comment]
Hi Thomas, both paths are in use and cannot be deleted. But I have deleted the `<defs></defs>` attribute.

**Comment by tauthomas01** [comment]
Hi @LaNeraNura , it's okay to keep the `path` for usage, but what I meant is removing the `id` attributes

[alt](https://screenshot.click/09-09-8nhc4-zwvqj.png)

**Comment by LaNeraNura** [comment]
Okay, I deleted the IDs.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @LaNeraNura ,

I still see some `id` instances that are not used. Can we make sure that all unused `id` attributes are removed?
For example `image902`.

The rest looks good.

**Comment by LaNeraNura** [comment]
Okay I removed the `image902`

**Comment by rahilvirani** [comment]
Closing the PR because it has been stale for a while. If the change is still relevant, please re-create the pull request using the new pull request template. We are sorry for the inconvenience.

**Pull request closed without merging**